src/gsi/read_fl_hdob.f90

```diff
@@ -1121,7 +1121,7 @@ subroutine read_fl_hdob(nread,ndata,nodata,infile,obstype,lunout,gstime,twind,si
               cdata_all( 1,iout)=woe                    ! wind error
               cdata_all( 2,iout)=dlon                   ! grid relative longitude             
               cdata_all( 3,iout)=dlat                   ! grid relative latitude                  
-              cdata_all( 4,iout)=dlnpsob                ! ln(surface pressure in cb)
+              cdata_all( 4,iout)=1000.                  ! ln(surface pressure in cb) !Since dlnpsob is not provided by SFMR, force it to be 1000hPa. Not used in setupspd.f90
```

Comment:
@XuLu-NOAA if it is not defined in this sub (and, supposedly, will not be used),  would it be better to assign r_missing to it?  This will help the code reading and avoid future use of this value by accident.

Reply:
Yes, @TingLei-NOAA , I think you are right, I will use the missing value